kafka-client/src/main/java/dev/responsive/db/CassandraWindowedSchema.java

```diff
@@ -98,7 +100,7 @@ public CassandraWindowedSchema(final CassandraClient client) {
 
   @Override
   public void create(final String name, Optional<Duration> ttl) {
-    // TODO: explore better data models for fetchRange/fetchAll
+    // TODO(window): explore better data models for fetchRange/fetchAll
```

Comment:
ultra-nit: usually this notation is used to make note of who owns the todo (e.g. TODO(sophie))

Reply:
yeah, I guess it's technically targeted at myself but I just wanted to mark everything I want to hit in the next window store PR. so it should be very temporary